### PR TITLE
Test/pagination super admin

### PR DESCRIPTION
**Pull request by hritikmunde**
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox

### CODE REVIEW

**Comment by hritikmunde** [comment]
@kevkevinpal , I'm having an issue with rendering MyTable and its css components. So I mocked the css using an empty file. Still, there is an error in HTML syntax, especially <h2>. If you have any suggestions on this I would appreciate that. I'm working on it for now.

**Comment by ecurrencyhodler** [comment]
You have some jest and prettier tests failing. Can you check them plz?

**Comment by hritikmunde** [comment]
Yes I'm working on it. There are quite few changes and html syntax errors to be addressed.

**Comment by hritikmunde** [comment]
Hey @ecurrencyhodler , can I get a code review? It has a lot of changes and reverts too but I managed to add tests in `TableComponent.spec.ts`

**Comment by ecurrencyhodler** [comment]
All tests passed. GJ with that. Yes let's get a code review.

File: frontend/app/src/pages/superadmin/tableComponent/index.tsx [hunk 1]
```diff
@@ -258,8 +257,13 @@ export const MyTable = ({
             <img src={copygray} alt="" width="16.508px" height="20px" />
             <LeadingTitle>
               {' '}
-              {bountiesLength}{' '}
-              <AlternativeTitle> {bountiesLength === 1 ? 'Bounty' : 'Bounties'}</AlternativeTitle>{' '}
+              {bounties.length}{' '}
```
Comment (kevkevinpal) [inline frontend/app/src/pages/superadmin/tableComponent/index.tsx:260]: I'm noticing these weird extra spaces `{' '}` ik it wasn't added in this PR but we should probably remove them

File: frontend/app/src/pages/superadmin/tableComponent/index.tsx [hunk 2]
```diff
@@ -258,8 +257,13 @@ export const MyTable = ({
...
+              <div>
+                <AlternativeTitle>
+                  {' '}
+                  {bounties.length === 1 ? 'Bounty' : 'Bounties'}
+                </AlternativeTitle>{' '}
```
Comment (kevkevinpal) [inline frontend/app/src/pages/superadmin/tableComponent/index.tsx:265]: here too I am noticing them

File: frontend/app/src/pages/superadmin/tableComponent/index.tsx (hunk 1 above)
Comment (hritikmunde) [reply frontend/app/src/pages/superadmin/tableComponent/index.tsx:260]: So you're suggesting I remove them? Is that is it then okay no issue

Comment (hritikmunde) [reply frontend/app/src/pages/superadmin/tableComponent/index.tsx]: Done

File: frontend/app/src/pages/superadmin/tableComponent/index.tsx (hunk 2 above)
Comment (hritikmunde) [reply frontend/app/src/pages/superadmin/tableComponent/index.tsx]: Done

**Comment by hritikmunde** [comment]
@kevkevinpal , any other discrepancies found? Or changes needed to be made. @ecurrencyhodler , resolved the changes.

**Review by kevkevinpal — APPROVED** [review]
Nice Work!

**Comment by ecurrencyhodler** [comment]
paid

**Pull request merged**